salt/metalk8s/container-engine/containerd/installed.sls

```diff
@@ -29,21 +31,33 @@ Install containerd:
     - require:
       - test: Repositories configured
       - metalk8s_package_manager: Install runc
+      - file: Create containerd service drop-in
+      - file: Configure registry IP in containerd conf
       {%- if grains['os_family'].lower() == 'redhat' %}
       - metalk8s_package_manager: Install container-selinux
       {%- endif %}
 
 Create containerd service drop-in:
   file.managed:
     - name: /etc/systemd/system/containerd.service.d/50-metalk8s.conf
-    - source: salt://{{ slspath }}/files/50-metalk8s.conf
+    - source: salt://{{ slspath }}/files/50-metalk8s.conf.j2
+    - template: jinja
     - user: root
     - group: root
     - mode: 0644
     - makedirs: true
     - dir_mode: 0755
-    - require:
-      - metalk8s_package_manager: Install containerd
+    - context:
+      environment: >-
+      {%- if proxies %}
+        NO_PROXY=localhost,127.0.0.1,{{ networks.values() | join(",") }}
```

Comment:
Shouldn't we allow to set *other* `NO_PROXY` values in configuration, similar to how HTTP/HTTPS proxies can be defined?

Reply:
Make sense, will do.

Reply:
done